test/e2e/resources/test-pod-no-requests.yaml

```diff
@@ -1,20 +1,12 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: cuda-vectoradd-no-requests
-  finalizers:
-  - org.instaslice/accelarator
+  name: vectoradd-no-req
 spec:
   restartPolicy: OnFailure
-  schedulingGates:
-  - name: org.instaslice/accelarator
   containers:
-  - name: cuda-vectoradd-no-requests
+  - name:  vectoradd-no-req
     image: "quay.io/tardieu/vectoradd:0.1.0"
```

Comment:
I think we can add an official NVIDIA `vectoradd` image for all our sample pods, e.g. `nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0`  (https://catalog.ngc.nvidia.com/orgs/nvidia/teams/k8s/containers/cuda-sample/tags)

Reply:
The last I checked there was issue running official image, let me create a Github issue and let's fix references in subsequent PR.

Reply:
created this #22